### PR TITLE
profile-summary save done

### PR DESCRIPTION
**Pull request by JRakin**

### CODE REVIEW

File: pages/api/users/models/UserSchema.ts [hunk 1]
```diff
@@ -1,13 +1,18 @@
 import { Schema, model, models } from 'mongoose';
 import bcrypt from 'bcryptjs';
 import { Gender } from '../../../../types/common.types';
-import { isEmail, isUrl } from 'js-string-helper';
+import { isEmail, isPhoneNumber, isUrl } from 'js-string-helper';
```
Comment (sifatul) [inline pages/api/users/models/UserSchema.ts:4]: @JRakin always keep same validation is backend and frontend. phone number validation in schema and formik is not matching. 

Better not to use validation for phone number in our case. we dont need

File: components/ProfilePageComponent/form/ProfileSummaryForm.tsx [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+                            />
+                            <div className={cx(ErrorMessageClass)}>
+                                <ErrorMessage name='phoneNumber' />
+                            </div>
+                        </div>
+                        <div className={cx(InputFieldContainer)}>
+                            <Field
+                                className={cx(InputField)}
+                                name='country'
+                                placeholder='Bangladesh'
```
Comment (sifatul) [inline components/ProfilePageComponent/form/ProfileSummaryForm.tsx:118]: @JRakin placeholder must not be your country

**Comment by JRakin** [comment]
@sifatul vaiya check please

File: components/ProfilePageComponent/form/ProfileSummaryForm.tsx [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+    country: Yup.string().required('Required'),
+    city: Yup.string().required('Required'),
+});
+
+const ProfileSummaryForm: React.FC<{ closeModal: () => void }> = ({ closeModal }) => {
+    const { _id = '' } = UseAppSelector(getUserState);
+
+    const formik = useFormik({
+        initialValues: {
+            profileHeading: '',
```
Comment (sifatul) [inline components/ProfilePageComponent/form/ProfileSummaryForm.tsx:67]: @JRakin how do you manage edit??

**Review by sifatul — CHANGES_REQUESTED** [review]

File: components/ProfilePageComponent/form/ProfileSummaryForm.tsx (hunk 3 above)
Comment (JRakin) [reply components/ProfilePageComponent/form/ProfileSummaryForm.tsx:67]: @sifatul ok got it i will update initValue

**Pull request merged**